tests/ffi/qm_nested/.testinfo.tmt

```diff
@@ -0,0 +1,7 @@
+Description: Check that QM executing well the nested container via podman.
+Owner: 
+RunFor: 
+RunTest: /bin/bash ./test.sh
+Path: /mnt/tests/tests/ffi/qm_nested
+Requires: 
+TestTime: 15m
```

Comment:
I think it's duration the name we are using for test time. Also the name we are using for the tmt file is main.tmt. 
```
tests/ffi/sysctl/main.fmf:3:duration: 20m
tests/ffi/dev_mem_not_present/main.fmf:3:duration: 10m
tests/ffi/qm-oom-score-adj/main.fmf:3:duration: 10m
tests/ffi/modules/main.fmf:3:duration: 20m
tests/ffi/agent-flood/main.fmf:3:duration: 10m
tests/ffi/deny_set_scheduler/main.fmf:3:duration: 10m
tests/ffi/tcp_max_syn_backlog/main.fmf:4:duration: 30m
tests/ffi/selinux/main.fmf:3:duration: 10m
tests/ffi/disk/main.fmf:3:duration: 10m
tests/ffi/memory/main.fmf:3:duration: 15m
tests/ffi/deny_sched_setattr/main.fmf:3:duration: 10m
```

In fact, it could be used any name but let's follow what we have to make easier for new people. The fields Owner, RunFor,  Path, Requires seems that can be removed.

Reply:
oh now I see you have main.fmf. Just question (not sure) can we merge this file with the main.fmf ?

Reply:
I removed the file. Its irrelevant to the test.